rdmo/core/filters.py

```diff
@@ -0,0 +1,30 @@
+from django.db.models import Q
+from django.utils.translation import get_language
+
+from rest_framework.filters import BaseFilterBackend
+
+from rdmo.core.utils import get_languages
+
+
+class SearchFilter(BaseFilterBackend):
+
+    def filter_queryset(self, request, queryset, view):
+        if view.detail:
+            return queryset
+
+        search = request.GET.get('search')
+        if search:
```

Comment:
if you like, you could do here:
```py
if not search:
    return queryset
....
```
so that the rest is less indented.
```
\
 \
++\
   \
~`~`\
     \
*&*&*&\
       \
++++++++\
         \
~`~`~`~`~`\
&*&*&*&*&*&\
.,.,.,.,.,.,.\
|
`|
```

Reply:
I am kind of used to a certain pattern when writing filters. If you have more than one query param it makes more sense.